tests/projects/test_virtualenv_projects.py

```diff
@@ -0,0 +1,41 @@
+import mlflow
+from unittest import mock
+from mlflow.utils.virtualenv import _create_virtualenv
+
+
+from tests.projects.utils import TEST_VIRTUALENV_PROJECT_DIR, TEST_VIRTUALENV_CONDA_PROJECT_DIR
+
+
+spy_on_create_virtualenv = mock.patch(
+    "mlflow.projects.backend.local._create_virtualenv", wraps=_create_virtualenv
+)
+
+
+@spy_on_create_virtualenv
+def test_virtualenv_project_execution(create_virtualenv_spy):
+    submitted_run = mlflow.projects.run(TEST_VIRTUALENV_PROJECT_DIR, env_manager="virtualenv")
+    submitted_run.wait()
+    create_virtualenv_spy.assert_called_once()
```

Comment:
Shall we assert the argument passed to `create_virtualenv_spy` are correct ?

Reply:
Which argument?

Reply:
The argument passed to mocked `mlflow.projects.backend.local._create_virtualenv`.

Reply:
I suggest to check all arguments except `capture_output`
`local_model_path, python_bin_path, env_dir, python_env, extra_env`